spec/mailers/notifier_spec.rb

```diff
@@ -39,4 +39,24 @@
       end
     end
   end
+
+  context "sms token change email" do
+    let(:mission) { create(:mission) }
+    let(:args) { [mission] }
+    let(:mail) { described_class.sms_token_change_alert(*args).deliver_now }
+
+    context "mission has coordinators" do
+      let!(:coordinator1) { create(:user, role_name: :coordinator, mission: mission) }
+      let!(:coordinator2) { create(:user, role_name: :coordinator, mission: mission) }
+      let!(:staffer) { create(:user, role_name: :staffer, mission: mission) }
+
+      it "should have all coordinators in to" do
+        expect(mail.to).to contain_exactly(coordinator1.email, coordinator2.email)
+      end
+
+      it "should have all coordinators in reply-to" do
+        expect(mail.reply_to).to contain_exactly(coordinator1.email, coordinator2.email)
```

Comment:
I would put both these assertions in one `it` block for efficiency since the fixtures are destroyed and rebuilt for each `it` block.